pages/templates/admin/base_site.html

```diff
@@ -0,0 +1,15 @@
+{% extends "admin/base.html" %}
```

Comment:
Ditto I hope this is right, took me a while to get this working.

Reply:
I'm getting these styles applied to every admin model. Very close, but it's a weird thing about django admin overrides.

We need to override just this template for this particular model and no others. The template that shows the list of models in the admin is called `change_list.html`, and it's one of five that can be overridden on a per model basis (https://docs.djangoproject.com/en/1.8/ref/contrib/admin/#templates-which-may-be-overridden-per-app-or-model).

To override this template, we need to place our new template file in the proper directory for django to recognize it as this model's override (https://docs.djangoproject.com/en/1.8/ref/contrib/admin/#overriding-admin-templates).

So, we need to copy the change_list.html template from Django 1.8 (https://github.com/django/django/blob/stable/1.8.x/django/contrib/admin/templates/admin/change_list.html), override it as we need (throw in the admin style you created), and Django will automatically apply it to the proper model if we have it in the right directory.

This new template should reside in `nominations/templates/admin/nominations/change_list.html`.

Let me know if you have questions.

Reply:
I considered altering the theme just for a specific model, but I thought applying the changes for the whole admin portal would be better UX since it's more consistent. Do you really think it's better to have a more compact theme just for a single model admin page, and have the default version elsewhere? I figured more compact everywhere in general would be better but either way is fine with me.

Reply:
I'm cool with this now that I think about it. We can change if we get negative feedback.